pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -1608,8 +1609,8 @@ func (pl *dynamicResources) reserveClaim(ctx context.Context, state *stateData,
 	if allocationPatch != "" {
```

Comment:
a bit odd that "reserveClaim" is called in PreBind instead of Reserve. Can you find a better name for the method? `bindClaim` would be a bit more accurate, analogous to binding a Pod to a node.

Reply:
Let's use `bindClaim`.